pkg/controller/tas/resource_flavor.go

```diff
@@ -203,3 +211,19 @@ func nodeBelongsToFlavor(node *corev1.Node, nodeLabels map[string]string, levels
 	}
 	return true
 }
+
+// checkNodeSchedulingPropertiesChanged checks if the node update affects TAS scheduling.
+func checkNodeSchedulingPropertiesChanged(oldNode, newNode *corev1.Node) bool {
+	oldCopy := oldNode.DeepCopy()
+	newCopy := newNode.DeepCopy()
```

Comment:
nit: do we need to deepcopy the entire node? Wondering if there's a subset of the node that scheduling cares about.